demos/demo-big-maps/src/demo_big_maps/models.py

```diff
@@ -5,23 +5,23 @@
 
 
 class TLD(Model):
-    id = fields.CharField(max_length=255, pk=True)
-    owner = fields.CharField(max_length=36)
+    id = fields.TextField(pk=True)
```

Comment:
According to tortoise code TextField is not recommended for pk
https://github.com/tortoise/tortoise-orm/blob/develop/tortoise/fields/data.py#L234

Reply:
Commited the following paragraph with explaination

>By default, Tortoise forbids index=True on TextField, and shows warning when it's pk=True.
> However, we only support SQLite and PostgreSQL and have no plans for others.
> For SQLite, there's only TEXT type. For PosrgreSQL, there's no difference between TEXT and VARCHAR
> except for length constraint. So we can safely use TEXT for both to avoid unnecessary schema changes.